auction-house/program/src/utils.rs

```diff
@@ -577,3 +625,39 @@ pub fn assert_valid_trade_state<'a>(
         _ => Err(ErrorCode::DerivedKeyInvalid.into()),
     }
 }
+
+pub fn assert_valid_auctioneer_and_scope(
```

Comment:
😁  Roger.

Reply:
This is actually a pain to unit test because of the AccountInfo business. None of the other assert methods are unit tested so unless you really think this needs to be done I'm going to skip it and rely on integration tests.